jetty-maven-plugin/src/it/javax-annotation-api/pom.xml

```diff
@@ -45,7 +45,12 @@
     </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
-      <artifactId>log4j-slf4j12</artifactId>
+      <artifactId>slf4j-log4j12</artifactId>
+      <version>@slf4j.version@</version>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>jul-to-slf4j</artifactId>
```

Comment:
This entry fixes a bad version mismatch for jul-to-slf4j that is brought in transitively by a different dependency